### ISSUE TITLE
Give a better explanation of the impact of forking in "What do I need to know if I fork the project?"

### ISSUE THREAD

**Issue by marctjones** [deptofdefense/code.mil#118]
I think the explanation in the FAQ on what happens if you fork the project stops a little short in the implications of licensing the U.S. Government's foreign copyrights. I would suggest that you beef the explanation up a little. 

Although a project consisting of only U.S. government works will not have any copyright protection so the license does not attach within the United States (or in any other jurisdiction that does not recognize copyright in U.S. government created works), downstream projects should understand that the Government is asserting that the LICENSE does apply in at least some foreign jurisdictions. Since many projects distribute code worldwide, you might want to suggest they keep that in mind if they include any of the code licensed in foreign jurisdictions. The explanation now might mislead a U.S. based project into thinking that they take the code free of all copyright restrictions, which is not really true.

For example: Government publishes project A, consisting only of contributions of government employees and asserts that it is licensed under the GPL in any foreign jurisdiction that recognizes copyright in U.S. government works. Assume the [Duchy of Grand Fenwick](https://en.wikipedia.org/wiki/The_Mouse_That_Roared) recognizes US Gov's copyright in it's original works of authorship. If Project B forks project A. Project B is based in the United States so takes Project A as being a public domain work. Project B does not include the foreign LICENSE or a copyright notice for the U.S. gov in project B for the portion of project A. Project B publishes on GitHub under a permissive license. A Fenwickian (F) downloads the code and includes it in another work. F includes project B in a closed source work (C) and distributes it within the Grand Duchy. F followed the permissive license.

Classic law school question: Who can sue who for what and who would win?

**Comment by shawoods** [comment]
@marctjones I totally agree the FAQ on forking is a little thin. I will ping this issue when I push a PR.

**Comment by jordangov** [comment]
@arichiv-usds I'm not sure if @marctjones' comment was fully addressed. Let's leave this open for a bit and see if he responds.

@marctjones Can you look at this [FAQ on forking](https://www.code.mil/frequently-asked-questions.html#what-do-i-need-to-know-if-i-fork-the-project) and let us know what you would change, or like added? If the current content is clear then we'll close this issue in a week or so.

**Comment by marctjones** [comment]
@shawoods @jordangov I think the first bullet point is misleading. It says "If the project includes no copyrighted code," but copyright is jurisdictionally based. So even though I am located in the United States and there are no copyrights in the code base here, but there does exist copyrights in Canada. The existence of copyrights depends on both who created the work and where a copy of the work is located. The existence of copyrights does not turn only on if a federal employee contributed the code since, if the federal employee's contribution otherwise meets the standards for copyrightability, copyrights do exist in some jurisdictions and the LICENSE has therefore attached in at least those jurisdictions. 

Not to over complicated this but Circular 1 from the copyright office says "Offering to distribute
copies or phonorecords to a group of people for purposes of further distribution, public perfor-
mance, or public display also constitutes publication."[1] So a open question someone should consider before using code.mil code on github for example is if they publishing the code in every country that has access to the code. This might depend on the law of the place of publication. 



Not exactly sure how to write this, but adding something:

 "Even if you are located in a jurisdiction that does not recognize copyright in federal government works, you should considering following the intended license since you will need to comply with the license if you ever distribute the code to jurisdiction that does. For example publishing the code on a public website accessible worldwide such as GitHub may count as a distribution to other jurisdictions that recognize copyrights in federal government works. You should also consider if you have an obligation to notify others who get the software from you if they have any obligations under the intended license if they distribute the software or use it in a jurisdiction that recognizes copyrights in federal government works."

[1] https://www.copyright.gov/help/faq-definitions.html

**Comment by jordangov** [comment]
@marctjones Thanks! I think that addition is useful, I'll try to get it into the site, might wordsmith it just a bit.

That said, you said the first bullet is misleading... thoughts on a better phrasing for it?

**Comment by marctjones** [comment]
@shawoods Yes, please wordsmith it. 

For the first paragraph, maybe something like: "If the project includes no copyrighted code _**in your jurisdiction**_ (so there have been no merged copyrighted contributions and your jurisdiction does not recognize copyright for U.S. Government-written work), then ‘LICENSE.md’ has not attached to the project.

**Comment by jordangov** [comment]
Cool, thanks Marc!

**Comment by GrooveCS** [comment]
Is this still an Open Issue?
